templates/springboot/http/pom.xml

```diff
@@ -18,9 +18,8 @@
 
   <properties>
     <java.version>11</java.version>
-    <spring-cloud.version>2021.0.1</spring-cloud.version>
-    <spring-cloud-function.version>3.2.3</spring-cloud-function.version>
-    <spring-native.version>0.11.3</spring-native.version>
+    <spring-cloud.version>2021.0.2</spring-cloud.version>
+    <spring-native.version>0.11.4</spring-native.version>
```

Comment:
Why was setting of `spring-cloud-function.version` removed?

Reply:
I am guessing that is coming from spring cloud now..

Reply:
so we can probably remove the version on that artifact, @trisberg can you confirm?

Reply:
Yes, the new spring-cloud 2021.0.2 BOM pulls in spring-cloud-function version 3.2.4

Reply:
Plus, setting `spring-cloud-function.version` property doesn't override what is in the BOM for spring-cloud 2021.0.1 since this isn't part of the spring-boot BOM, you would actually have to define the dependency explicitly like

```
			<dependency>
				<groupId>org.springframework.cloud</groupId>
				<artifactId>spring-cloud-function-dependencies</artifactId>
				<version>3.2.3</version>
				<type>pom</type>
				<scope>import</scope>
			</dependency>
```

Reply:
Looks like I missed some dependency settings, added another commit to fix this